_episodes/02-loop.md

```diff
@@ -141,6 +141,14 @@ for variable in collection:
 ~~~
 {: .language-python}
 
+> ## What is the ":" for?
+>
+> One thing you may have noticed is that our `for` statement ends with a `:`. You actually will see the `:` regularly in Python, and
```

Comment:
1.
> You might have noticed that ...

seems easier to say than

>  One thing you may have noticed is that

2. 
> our `for` statement

it is not "our". This are just "`for` statements"